client/Containers/UserDashboard.jsx

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+import Login from './Login';
+
+// Filter users based on 'isAdmin' attribute
+
+class UserDashboard extends Component {
+  state = {
+    auth: {},
+    existingUsers: [],
+  };
+
+  logout = () => {
+    window.localStorage.removeItem('token');
+    this.setState({ auth: {} });
+  };
+
+  attemptTokenLogin = async () => {
+    const token = window.localStorage.getItem('token');
+    if (token) {
+      const response = await axios.get('/api/auth', {
+        headers: {
+          authorization: token,
+        },
+      });
+      const existingUsers = await axios.get('/api/users');
+      // for future referenc: {headers: { authorization: token }} may be required for loading user orders
+      this.setState({ auth: response.data, existingUsers: existingUsers.data });
```

Comment:
Totally optional, but we could destructure `data` from each get request and rename them to correspond to state, so the `this.setState()` call can just use the variable names. Like this:
```javascript
const { data: auth } = await axios.get('/api/auth', ...);
const { data: existingUsers } = await axios.get('/api/users');
this.setState({ auth, existingUsers });
```